src/Common/src/Interop/Windows/mincore/Interop.CreateFile.cs

```diff
@@ -18,5 +19,18 @@ internal partial class mincore
             System.IO.FileMode dwCreationDisposition,
             int dwFlagsAndAttributes,
             IntPtr hTemplateFile);
+
+        internal static SafeFileHandle CreateFile(
```

Comment:
I'll check what happens for short and long paths. I don't think the handle is connected in any way with the original query.